ga4gh/datamodel/variants.py

```diff
@@ -410,6 +445,26 @@ def convertVariant(self, record, callSetIds):
         variant.id = self.getVariantId(variant)
         return variant
 
+    def getVariant(self, compoundId):
+        if compoundId.referenceName in self._chromFileMap:
+            varFileName = self._chromFileMap[compoundId.referenceName]
+        else:
+            raise exceptions.ObjectNotFoundException(compoundId)
+        start = int(compoundId.start)
+        # fetch start is exclusive, not inclusive
```

Comment:
Does this comment still hold? You don't seem to be doing anything about it if it does...